client/src/components/ElementEditor/ElementList.js

```diff
@@ -48,7 +48,7 @@ class ElementList extends Component {
     }
 
     return blocks.map((element) => (
-      <div>
+      <div key={element.ID}>
```

Comment:
I'm curious about whether giving it the same key as its child element might cause rendering issues... I don't know enough about how this works in React though

Reply:
In this case I think we can remove the key from the child component. I'm not 100% on this either but I'm pretty sure it's only used for the top level dom elements within a loop.

Reply:
"Adjacent JSX elements must be wrapped in an enclosing tag" 😉